torch/_inductor/codegen/common.py

```diff
@@ -805,8 +819,21 @@ class CSEProxy:
             @staticmethod
             def __getattr__(name):
                 def inner(*args, **kwargs):
+                    # TritonTemplateKernel has no current_node
+                    buf_bounds = ValueRanges.unknown()
+                    if hasattr(V.interpreter, "current_node"):
+                        fx_node = V.interpreter.current_node
+                        buf_bounds = self.bounds.get(fx_node, ValueRanges.unknown())
```

Comment:
worth turning `bounds.get(fx_node, ValueRanges.unknown())` into an API ?

Reply:
I think not yet. This may be interesting to discuss if we lift the analysis to the whole graph (and the backward graph). At the moment `self.bounds` is not the main entrypoint. `csevar.bounds` is.